dcos_tunnel/cli.py

```diff
@@ -75,6 +76,8 @@
     --host=<host>
         Manually specify which host to connect to (it will override attempts to
         detect which host to connect to)
+    --docker=<docker-command>
+        Docker client to run [default: docker]
```

Comment:
We're already setting a hard coded default right now. This just maintains that default.